### ISSUE TITLE
NTR:Bilateral frontoparietal polymicrogyria

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4564]
label:Bilateral frontoparietal polymicrogyria (synonym: ) ID HP:0007095 parent Polymicrogyria def Symmetric generalized polymicrogyria with decreasing anterior-posterior gradient, most prominent in frontoparietal cortex. The imaging pattern can mimic frontal-predominant cobblestone malformation. [20301504] COST:neuromig

**Comment by pnrobinson** [comment]
@Renske-Oegema Is it sufficient to use the modifier terms for this? https://hpo.jax.org/app/browse/term/HP:0012833 the HPO tries to avoid having separate terms for uni/bilateral unless there is an important distinction for differential diagnosis
